.husky/pre-commit

```diff
@@ -1 +1,2 @@
-npx lint-staged --verbose
+npx lint-staged --verbose 
+npx prettier -w e2e/main.e2e.test.ts
```

Comment:
This `-w` exclusion ought to go in `.linstagedrc.json`, that way we won't run prettier against the whole codebase every commit.